samples/compare/src/test/kotlin/okhttp3/compare/ApacheHttpClientTest.kt

```diff
@@ -28,17 +28,17 @@ import org.junit.jupiter.api.Test
  * Apache HttpClient 5.x.
  *
  * https://hc.apache.org/httpcomponents-client-5.0.x/index.html
+ *
+ * Baseline test if we ned to validate OkHttp behaviour against other popular clients.
  */
-class ApacheHttpClientTest(
-  private val server: MockWebServer
-) {
+class ApacheHttpClientTest {
```

Comment:
This change was accidental because I made the changes on old copy of master before yours.  But leaving for discussion.